README.md

```diff
@@ -22,9 +22,11 @@ You can find more information about this design pattern here:
 * [Page Objects - Selenium wiki](https://code.google.com/p/selenium/wiki/PageObjects)
 * [PageObject - Martin Fowler](http://martinfowler.com/bliki/PageObject.html)
 
-## Documentation
+## Community
 
-Check the [site](http://ember-cli-page-object.js.org/) for full documentation.
+Let's work together to improve this addon!
+
+We're on the [official Ember slack](https://ember-community-slackin.herokuapp.com/), join to our channel #ember-cli-page-object or [open an issue on Github](https://github.com/san650/ember-cli-page-object/issues) to request features, report bugs or just to ask any question.
```

Comment:
You can find us on the [official community](https://ember-community-slackin.herokuapp.com/), join the `ember-cli-page-object` channel or [open an issue on Github](https://github.com/san650/ember-cli-page-object/issues) to request features, report bugs or just to ask any question.